src/HotChocolate/Core/src/Types/Types/Relay/GlobalIdInputValueFormatter.cs

```diff
@@ -5,6 +5,7 @@
 using System.Linq.Expressions;
```

Comment:
There is also a implementation of this in filtering that we would have to change ;)